### DISCUSSION TITLE
Statistics of yanked crate version

### DISCUSSION THREAD

**Discussion by gnzlbg** [rust-lang/crates.io#4338] [Feature Requests]
Crates with many yanked versions _might_ be unreliable. It would be neat if crates.io could display:

* number of yanked versions
* ratio of: number of yanked versions / total number of released versions

I think this is a useful metric because for crates with a high ratio of yanked versions the publisher might be unreliable, and dependent crates might want to choose a  concrete version instead of relying on globs.

**Comment by sgrif** [comment]
Do you have a specific place in the UI where you think it would make sense to display this?

**Comment by gnzlbg** [comment]
For example under "Stats overview" we could have another column with "Yanked versions: #COUNT" .
